source/server/admin/utils.cc

```diff
@@ -46,9 +46,21 @@ absl::Status histogramBucketsParam(const Http::Utility::QueryParamsMulti& params
   return absl::OkStatus();
 }
 
+// Helper method to get a query parameter.
+// Returns the first value for that query parameter, unless that value is empty.
+// In that case, it returns nullopt.
+absl::optional<std::string> notEmptyQueryParam(const Http::Utility::QueryParamsMulti& params,
```

Comment:
nit: nonEmpty?

I'm OK if you decide you prefer notEmpty but to me, "not" suggests this is returning a bool.